### PR TITLE
Maint syntax cleanup

### PR DESCRIPTION
**Pull request by vEpiphyte**
- Use one style of comments (two spaces)
- Add all deprecation warnings and fix them where present
- Check for bad indents

### CODE REVIEW

File: synapse/lib/types.py [hunk 1]
```diff
@@ -1598,7 +1598,7 @@ def _normPyStr(self, valu):
 
         return norm, info
 
-taxonre = regex.compile('\w+')
+taxonre = regex.compile('\\w+')
```
Comment (rakuy0) [inline synapse/lib/types.py:1601]: Uhh, isn't this changing the meaning of the regex to mean something very different?

File: synapse/lib/types.py [hunk 2]
```diff
@@ -1678,7 +1678,7 @@ def _normPyStr(self, text):
         toks = text.strip('#').split('.')
         return self._normPyList(toks)
 
-tagpartre = regex.compile('\w+')
+tagpartre = regex.compile('\\w+')
```
Comment (rakuy0) [inline synapse/lib/types.py:1681]: Same as the other regex comment

File: synapse/models/gov/intl.py [hunk 3]
```diff
@@ -5,7 +5,7 @@ class GovIntlModule(s_module.CoreModule):
     def getModelDefs(self):
         modl = {
             'types': (
-                ('iso:oid', ('str', {'regex': '^([0-2])((\.0)|(\.[1-9][0-9]*))*$'}), {
+                ('iso:oid', ('str', {'regex': '^([0-2])((\\.0)|(\\.[1-9][0-9]*))*$'}), {
```
Comment (rakuy0) [inline synapse/models/gov/intl.py:8]: Similar to before, isn't this changing the meaning of the regex from "match a dot" to "match a backslash and then any character"?

File: synapse/lib/types.py (hunk 1 above)
Comment (vEpiphyte) [reply synapse/lib/types.py:1601]: These are using deprecated escape sequences in string types which will eventually become a python syntax error.

File: synapse/lib/types.py (hunk 2 above)
Comment (vEpiphyte) [reply synapse/lib/types.py:1681]:
```
>>> '\w+' == '\\w+'
True
```

File: synapse/models/gov/intl.py (hunk 3 above)
Comment (vEpiphyte) [reply synapse/models/gov/intl.py:8]:
```
>>> so = '^([0-2])((\.0)|(\.[1-9][0-9]*))*$'
>>> sn = '^([0-2])((\\.0)|(\\.[1-9][0-9]*))*$'
>>> so == sn
True
>>> so
'^([0-2])((\\.0)|(\\.[1-9][0-9]*))*$'
>>> sn
'^([0-2])((\\.0)|(\\.[1-9][0-9]*))*$'
```

File: synapse/lib/types.py (hunk 1 above)
Comment (vEpiphyte) [reply synapse/lib/types.py:1601]:
```
>>> '\w+' == '\\w+'
True
```

**Review by rakuy0 — APPROVED** [review]

**Pull request merged**